processor/README.md

```diff
@@ -235,15 +235,20 @@ examples on using the processor.
 The batch processor accepts spans and places them into batches grouped by node
 and resource. Batching helps better compress the data and reduce the number of
 outgoing connections required to transmit the data. This processor supports
-both size and time based batching. Please refer to
-[config.go](batchprocessor/config.go) for the config spec.
+both size and time based batching.
+
+It is highly recommended to configure the batch processor on every collector.
```

Comment:
I would say that it should come before the queued_retry and after any processor that can remove data from the pipeline (eg.: the probabilistic sample processor) since we don't want the batcher to be handling data that will be dropped.

Reply:
ah, good call, let me address